### ISSUE TITLE
Problem runing trade.py

### ISSUE THREAD

**Issue by budaspriestsl** [LeaveMyYard/BitmexGridTrader#2]
Hi,
I'm taking this message running trade.py after instalation of requeriments.txt

C:\Users\Tester\Downloads\BitmexGridTrader-master>python trade.py
Traceback (most recent call last):
  File "trade.py", line 27, in <module>
    main()
  File "trade.py", line 21, in main
    window = MainWindow(loop)
  File "C:\Users\Tester\Downloads\BitmexGridTrader-master\sourse\ui\mainwindow.py", line 73, in __init__
    self.handle = BitmexExchangeHandler(*self.current_settings.get_current_keys())
TypeError: Can't instantiate abstract class BitmexExchangeHandler with abstract methods get_symbols_data

I try using linux and i'm taking the same issue, please can you help me?

**Comment by m4ryou5h** [comment]
I have the same issue. Running python 3.8.3

**Comment by m4ryou5h** [comment]
I tried adding API keys but it didn't help to start it.
I guess this is hardcoded for mainnet as I couldn't find switch for testnet?

**Comment by caressgents** [comment]
@LeaveMyYard I am getting the exact same error as above

**Comment by LeaveMyYard** [comment]
As I answered in other issue #1 , it is a bug in my other library, that is used in this project and I will fix it in 24 hours.

As soon as it will be fixed, I will write here.

**Comment by LeaveMyYard** [comment]
As I wrote in #1

I've updated [CryptoFuturesPy](https://github.com/LeaveMyYard/CryptoFuturesPy) to 0.4.3, found and fixed some connector bugs.
Also, updated this project to support new features of this library in 6a8b9c1e8266909ecbe2b082d8162da77414cb31
Now, it should start and work fine, just run `pip install -r requirements.txt` or just `pip install crypto-futures-py==0.4.3`

Try to update a project files and launch it.

*Keep in mind, that there could exist some other bugs. For example, "Stop bot" button will not stop the grid from being updated, so I propose to close the bot window entirely to stop the bot. Also, always check your orders directly on bitmex to be sure that nothing went wrong.*

Feel free to write here or to open a new issue if something will still work wrong :)
